pom.xml

```diff
@@ -393,67 +338,13 @@ under the License.
 				<version>3.3.2</version>
 			</dependency>
 
-			<dependency>
-				<groupId>org.xerial.snappy</groupId>
-				<artifactId>snappy-java</artifactId>
-				<version>1.1.8.3</version>
-			</dependency>
-
-			<dependency>
-				<groupId>org.lz4</groupId>
-				<artifactId>lz4-java</artifactId>
-				<version>${lz4.version}</version>
-			</dependency>
-
-			<dependency>
-				<groupId>com.github.oshi</groupId>
-				<artifactId>oshi-core</artifactId>
-				<version>3.4.0</version>
-			</dependency>
-
-			<!-- We no longer align the avro version with the version bundled in Hadoop.
-			 Users might need to downgrade the avro version for a particular Hadoop version. -->
-			<dependency>
-				<groupId>org.apache.avro</groupId>
-				<artifactId>avro</artifactId>
-				<version>${avro.version}</version>
-			</dependency>
-
 			<!-- For dependency convergence -->
 			<dependency>
```

Comment:
Nit: Do we still need hamcrest? I hoped all assertions are migrated to assertJ by now.

Reply:
I still see Hamcrest being used in the Java code unfortunately...

Reply:
@alpreu Do you know if we could fix this?

Reply:
Yes, I'll fix it

Reply:
https://github.com/apache/flink-connector-elasticsearch/pull/17